workflow-templates/gradle.properties.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Build using Gradle",
+    "description": "Build using Gradle, by default version 5.2.1 is used.",
+    "iconName": "gradle"
```

Comment:
As per https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template:
-  proposing to add `Gradle` and/or `Java` as the `Categories` - see https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
- can we make assumptions about the `filePatterns` e.g. `build.gradle` or `*.gradle` in the top-level folder?
filePatterns

Reply:
build.gradle is in the top-level folder of our repositories.

Reply:
added ``Categories`` and ``filePatterns``